meson.build

```diff
@@ -205,7 +205,17 @@ endforeach
 # Set excutable rpaths for non standard prefixes
 
 if get_option('set_rpath').enabled() or (get_option('set_rpath').auto() and get_option('prefix') != '/usr')
-    install_rpath = get_option('prefix') / get_option('libdir')
+    if host_machine.system() == 'darwin' and get_option('build_macos_app') == true
+        macos_app_path = get_option('prefix')
+        macos_app_contents = macos_app_path / 'Contents'
+
+        macos_app_exe = macos_app_contents / 'MacOS'
+        macos_app_resources = macos_app_contents / 'Resources'
+
+        install_rpath = macos_app_path
```

Comment:
These variables are primarily used in the install section below. I think we should move them there.

```suggestion
```